.github/workflows/clients-java-build.yml

```diff
@@ -22,12 +19,18 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+      with:
```

Comment:
Do we need the `fetch-depth` set to 0 here?

Reply:
We need to auto-increment the version number using the commits count. 
By default `actions/checkout@v3` only fetches the last commit, so without that setting `git rev-list --count HEAD` always returns `1`.